SQL/New_patches/2019-11-27-RemoveOldConfigSettings.sql

```diff
@@ -0,0 +1,4 @@
+DELETE FROM ConfigSettings WHERE Name='Logs';
+DELETE FROM ConfigSettings WHERE Name='css';
```

Comment:
This should not be removed. Your code changes to remove this are making it far more complicated. There's now multiple sources scattered across the code base referencing the file instead of a single source of truth.

Reply:
> CSS -- A FIXME in the middleware says that it's not actually supported

I wrote this in the original issue a long time ago. Do you remember this? I feel like you're probably the author.

I don't mind keeping it if it's used.

Reply:
I don't mind the feature of changing the filename/the config setting being removed (and it's possible that it doesn't work if it's changed, I haven't checked..), it's the fact that this PR adds multiple scattered constants to replace what was previously a single well-defined canonical value that I think is a step backwards and I'd rather have a useless config than the maintenance nightmare of keeping them in sync.

I don't currently see any FIXME referencing it in the Middleware directory.

Reply:
OK this part may be outdated. I'll revert these changes.

Reply:
Found it:

https://github.com/aces/Loris/blob/6190c99098bd2b40892b8f72bfd85606ed381264/src/Middleware/UserPageDecorationMiddleware.php#L131-L135